tools/dotnet-linker/Steps/ConfigurationAwareStep.cs

```diff
@@ -1,9 +1,20 @@
+using System;
+using System.Collections.Generic;
 using Mono.Linker.Steps;
 
+using Xamarin.Bundler;
+
 namespace Xamarin.Linker {
 	public abstract class ConfigurationAwareStep : BaseStep {
 		public LinkerConfiguration Configuration {
 			get { return LinkerConfiguration.GetInstance (Context); }
 		}
+
+		protected void Report (List<Exception> exceptions)
+		{
+			// Maybe there's a better way to show errors that integrates with the linker?
```

Comment:
some API was added in the linker (not yet released, or available in the facade, when I worked on this)

Reply:
Yes, I'll look into that later, and fix it in a different PR.